


<!DOCTYPE html>
<html lang="en">
<head>
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-108098699-1"></script>
  <script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-108098699-1');
  </script>

  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="">
  <meta name="author" content="">
  <link rel="shortcut icon" href="../../assets/ico/favicon.ico">

  <title>Dipti D. Nayak</title>

  <!-- Bootstrap core CSS -->
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
  <link href="css/bootstrap.css" rel="stylesheet">
  <link href='https://fonts.googleapis.com/css?family=Gentium+Basic' rel='stylesheet' type='text/css'>
<link href='https://fonts.googleapis.com/css?family=Andada' rel='stylesheet' type='text/css'>
<link href='https://fonts.googleapis.com/css?family=Average' rel='stylesheet' type='text/css'>


  <!-- Custom styles for this template -->
  <link href="css/offcanvas.css" rel="stylesheet">
  <!--<link href="css/ap.css" rel="stylesheet">-->


</head>


<body>
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script> 
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  <script src="js/bootstrap.js"></script>

  <script>$(document).ready(function() {
    $("[id^=content]").hover(function() {
        $(this).addClass('transition');
    }, function() {
        $(this).removeClass('transition');
    });
});
</script>


<!--<img id="content" src="http://upload.wikimedia.org/wikipedia/en/7/78/Small_scream.png">-->

  

   <div class="navbar navbar-fixed-top navbar-inverse" role="navigation">
        <div class="container">
          <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
              <span class="sr-only">Toggle navigation</span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="index.html#">Dipti Nayak</a>
          </div>
          <div class="collapse navbar-collapse">
            <ul class="nav navbar-nav">
              <li><a href="index.html#">Home</a></li>
              <li><a href="index.html#bio">Bio</a></li>
              <li><a href="index.html#research">Research</a></li>
              <li><a href="index.html#outreach">Outreach</a></li>
              <li><a href="index.html#service">Service</a></li>              
              <li><a href="cv.pdf">CV</a></li>

            </ul>
      </div>
    </div>
  </div>

  <div class="container">

    <div class="row row-offcanvas row-offcanvas-right">

      <div class="col-xs-12 col-sm-9">
          <p class="pull-right visible-xs">
            <button type="button" class="btn btn-primary btn-xs" data-toggle="offcanvas">Other Info</button>
          </p>
          <div class="jumbotron">
            
            <div class="row">
              <div class="col-md-4">
               <img src="images/dipti.jpg" class="img-responsive img-rounded" style="margin-top: 10px; margin-bottom: -25px;">  
             </div> <!-- /span2 -->
             <div class="col-md-8">
              <p>
                I am a <a href="https://www.simonsfoundation.org/life-sciences/simons-postdoctoral-fellowships-in-life-sciences/">Life Sciences Research Foundation Postdoctoral Fellow</a>, and a <a href="http://www.igb.illinois.edu/involve/become-a-fellow">Carl R. Woese Institute for Genomic Biology Fellow</a>
                at the University of Illinois.  <br/><br/>

                My research falls at the intersection of microbial physiology and evolution: I employ techniques
                from both fields to understand the fundamental biology of methanogenic and methanotrophic <a href="https://en.wikipedia.org/wiki/Archaea">Archaea</a>, 
                an important class of organisms within the <a href="https://en.wikipedia.org/wiki/Three-domain_system">third domain of Life</a>, 
                with wide-ranging implications in climate change, renewable fuels, and human health.
                </p>
            </div>
          </div>
      </div>


      <a name="bio"></a> 
      <div class="row">
        <div class="col-md-12">
          <b> Update (February 2019): I will be joining the <a href="https://mcb.berkeley.edu/">Molecular and Ceulluar Biology Department</a> at UC Berkeley as a tenure-track assistant professor starting July 1 2019. If you are interested in applying to the lab please feel free to reach out me at dipti.nayak[at]gmail.com <b>
          <h2> About Me </h2>
       I am a Life Science Research Foundation Fellow courtesy of the <a href="https://www.simonsfoundation.org/life-sciences/simons-postdoctoral-fellowships-in-life-sciences/">Simons Foundation</a> (awarded nationally to ~30 from over 1000 candidates in all
          fields of life sciences), and a <a href="http://www.igb.illinois.edu/involve/become-a-fellow">Carl R. Woese Institute for Genomic Biology Fellow</a> at the University of Illinois, working with Prof. <a href="https://mcb.illinois.edu/faculty/profile/metcalf/">William W. Metcalf</a>. I received my PhD 
          from the <a href="https://oeb.harvard.edu/">Department of Organismic and Evolutionary Biology </a> 
          at Harvard University, under the direction of Prof. <a href="https://marxlab.org/">Christopher J. Marx</a> in 2014, and a Masters degree from 
                the <a href="https://cee.stanford.edu/people/">Department of Civil and Environmental Engineering</a> at Stanford University in 2009. 
       My PhD research focused on using experimental evolution as a forward genetic screen to study metabolism and stress response in microorganisms that grow on reduced single-carbon compounds (methylotrophs). My postdoctoral research has focused on the physiology and evolution of methanogenic archaea, employing Cas9-based genome editing tools &mdash; <a href="http://www.cell.com/trends/microbiology/fulltext/S0966-842X(17)30072-0">the first successful application of Cas9-mediated genome editing in a member of the archaeal domain</a>. 
        </div>
      </div>
      <div class="row">
        <div class="col-md-12">
          <h2> Motivation and Research Interests </h2>
          <p>
           Methane is a tremendous societal burden and a valuable resource: on the one hand, it is a potent greenhouse gas, and on the other, it is a cheap, renewable fuel with immense potential for manufacturing value-added chemicals. Technological advances are needed at both ends: for mitigating atmospheric emissions, as well as converting organic-rich effluents to methane. 
           At the heart of these problems, and their solutions, lie <a href="https://en.wikipedia.org/wiki/Methanogen">methanogenic</a> and <a>methanotrophic</a> <b>archaea</b>: the primary producers and consumers of methane respectively. 
           </p><p>
          Despite being ubiquitous, archaea are among the least-characterized life forms on Earth. A strictly anoxic lifestyle and slow growth have particularly impeded comprehensive studies of methanogenic and methanotrophic archaea; therefore our understanding of these pivotal microorganisms is severely deficient. By integrating an evolutionary framework with a rapid, multiplexed genome editing toolkit my research aims to overcome these roadblocks. In on-going work, I am designing a wide-range of experimental platforms to test prevailing hypotheses, as well as discover novel and unique features of methanogenic and methanotrophic archaea. 
          </p><p>
          <b><i>Broadly, I work at the intersection of physiology and evolution to understand the fundamental and distinctive biology of methanogenic and methanotrophic archaea, from a cellular, organismal, and community perspective</i></b>. This knowledge is key to designing optimized solutions for aforementioned environmental and biotechnological problems.
          </p>
        </div>
      </div>
      <!--
      <a name="research"></a>
      <div class="row">
        <div class="col-md-12">
          <h2> Research Projects </h2>
          <h4 class="media-heading"><b>CRISPR Blah in Blah</b></h4>
                  <p> I did some blah when blahing 
                  </p>
          <h4 class="media-heading"><b>Evolutionary Blah in Blah </b></h4>
                  <p> I did some blah when blahing 
                  </p>
        </div>
      </div>    
      -->
      

      <div class="row">
        <div class="col-md-12">
          <h2> Publications </h2>
          <ul>
            <li> <!--<img id="content" src="images/elife.png" width="150" style="border:1px solid grey;vertical-align:text-bottom">-->D. D. Nayak, N. Mahanta, D.A. Mitchell, W.W. Metcalf, 
            <a href="https://elifesciences.org/articles/29218">Post-translational thioamidation of methyl-coenzyme M reductase, a key enzyme in methanogenic and methanotrophic Archaea</a>, 
            <b>eLife</b>, doi: 10.7554/eLife.29218, 2017. 
            <button type="button" class="btn-xs" data-toggle="collapse" data-target="#demo1"">More Info</button>
            <div id="demo1" class="collapse">
             <ul>
            <li>	Methyl-coenzyme M reductase (MCR) is found in methanogenic and methanotrophic archaea and catalyzes the reversible production and consumption of methane </li>
<li>	A highly unusual thioglycine modification, in which the peptide amide bond is converted to a thioamide, is present in the α-subunit of all characterized MCRs and is implicated to play a critical role in catalysis </li>
<li>	Using our Cas9-mediated genome editing tools, we show that mutants of the methanogen <i>Methanosarcina acetivorans</i> with deletions in the <i>ycaO-tfuA</i> operon are viable and lack the thioglycine modification in MCR </li>
<li>	Our analyses support a role for thioglycine in stabilizing the protein secondary structure near the active site of MCR </li>
                                            </ul>
            </div>
            </li><br/>
            <li><!--<img id="content" src="images/pnas.gif" width="100" style="border:1px solid grey; vertical-align:text-bottom">--> D. D. Nayak, W.W. Metcalf, <a href="https://www.ncbi.nlm.nih.gov/pubmed/28265068">Cas9-mediated genome editing in the methanogenic archaeon <i>Methanosarcina acetivorans</i></a>, <b>Proceedings of the National Academy of Sciences (PNAS)</b>, doi: 10.1073/pnas.1618596114, 2017. 
            <button type="button" class="btn-xs" data-toggle="collapse" data-target="#demo2"">More Info</button>
            <div id="demo2" class="collapse">
              <ul>
                <li> We report the development of the first Cas9-mediated genome editing tool for an archaeon, the slow-growing methanogen <i>Methanosarcina acetivorans</i> </li> 
<li>	Introduction of both insertions and deletions by homology directed repair (HDR) is efficient and precise (~ 20% of transformation efficiency) without any detectable off-target activity </li>
<li>	Multiple single guide RNAs (sgRNAs) can be expressed in the same transcript and allows simultaneous construction of double mutants with high efficiency </li>
<li>	Co-expression of the non-homologous end joining (NHEJ) machinery from the closely related archaeon, <i>Methanocella paludicola</i>, allows for efficient Cas9-mediated genome editing without the need for a repair template </li>
              </ul>
            </div>
            <br/>
            Press and other mentions:
            <ul>
              <li> <a href="http://www.cell.com/trends/microbiology/fulltext/S0966-842X(17)30072-0">"Finally, Archaea Get Their CRISPR-Cas Toolbox"</a>, spotlight article in Trends in Microbiology, Volume 25, Issue 6, June 2017, Pages 430-432.  Abstract: <i>"The majority of archaea encode CRISPR-Cas systems but only a few CRISPR-Cas-based genetic tools have been developed for organisms from this domain. Nayak and Metcalf have harnessed a bacterial Cas9 protein for genome editing in Methanosarcina acetivorans, enabling efficient gene deletion and replacement."</i>
              </li>
              <li> <a href="http://igb.illinois.edu/news/new-tool-genetically-engineering-oldest-branch-life">"A New Tool for Genetically Engineering the Oldest Branch of Life"</a>, article by Institute of Genomic Biology, covered by Department of Energy (DOE) University Research News, Phys.org, Science Daily.
              </li>
            </ul>
            </li>
            
            <br/>
            <li> <!--<img id="content" src="http://www.cell.com/cms/attachment/2063769399/2065760075/gr1.jpg" width="100" style="border:1px solid grey;vertical-align:text-bottom">-->D. D. Nayak, D. Agashe, M-C Lee, C.J. Marx, <a href="https://www.ncbi.nlm.nih.gov/pubmed/27212407">Selection maintains apparently degenerate metabolic pathways
due to tradeoffs in using methylamine for carbon versus nitrogen</a>, <b> Current Biology</b>, Cell Press, Volume 26,
Issue 11, pp 1416–1426, 2016.  
            <button type="button" class="btn-xs" data-toggle="collapse" data-target="#demo3"">More Info</button>
            <div id="demo3" class="collapse"> <ul>
           <li> Methylotrophic bacteria grow on reduced single-carbon compounds and commonly encode two functionally degenerate non-orthologous pathways for methylamine oxidation </li>
<li>	We show that, despite catalyzing the same reaction, each pathway has a distinct function in the methylotroph <i>Methylobacterium extorquens</i> AM1 </li>
<li>	The periplasmic route for methylamine oxidation, mediated by methylamine dehydrogenase, enables optimal utilization of methylamine as a growth substrate </li>
                                            <li> The cytoplasmic route for methylamine oxidation, mediated by the <i>N</i>-methylglutamate pathway, leads to efficient nitrogen assimilation from methylamine, especially at low concentrations </li>
<li>	Our analyses indicate that tradeoffs between cellular localization and ammonium toxicity select for this functional degeneracy </li>
</ul>
            </div>
            <br/>
            Press Mentions:
            <ul>
            <li> <a href="http://www.uidaho.edu/sci/biology/news/features/2016/evolving-an-explanation">"Evolving an Explanation"</a>, article by the University of Idaho Press.
            </li>
            </ul>
            </li><br/>
            <li> <!--<img id="content" src="http://www.mdpi.com/microorganisms/microorganisms-03-00060/article_deploy/html/images/microorganisms-03-00060-g003.png" width="100" style="border:1pt solid grey; vertical-align:text-bottom">-->  D. D. Nayak, C.J. Marx, <a href="http://www.mdpi.com/2076-2607/3/1/60">Experimental horizontal gene transfer of methylamine dehydrogenase mimics prevalent
            exchange in nature and overcomes the methylamine growth constraints posed by the sub-optimal <i>N</i>-methylglutamate
            pathway</a>, <b> Microorganisms</b>, Volume 3, Issue 1, pp 60–79, 2015. 
            <button type="button" class="btn-xs" data-toggle="collapse" data-target="#demo4"">More Info</button>
            <div id="demo4" class="collapse">
              <ul>
            <li>	Methylotrophic bacteria that grow on the reduced single-carbon compound methylamine encode either one or both of the characterized metabolic modules for methylamine oxidation </li> 
                <li>	Phylogenetic analyses demonstrate that the <i>N</i>-methylglutamate (NMG) pathway is more widely distributed and follows the vertical phylogeny of methylotrophs. In contrast, the distribution of methylamine dehydrogenase (MaDH) is patchy and indicative of numerous horizontal gene transfer (HGT) events. </li> 
                <li> Phenotypic analyses of <i>Methylobacterium extorquens </i>strains indicates that the presence of MaDH is correlated with significantly higher fitness on methylamine </li>
                <li> Mimicking a HGT event in the laboratory, by introducing MaDH in <i>M. extorquens</i> PA1, a strain that only encodes the NMG pathway, increases fitness by 11-fold instantaneously </li>
<li>	In contrast, even after 150 generations of experimental evolution on methylamine the fitness of <i>M. extorquens</i> PA1 does not increase </li>
<li>	Our analyses indicate that HGT can overcome adaptive constraints posed by sub-optimal metabolic modules </li>
              </ul>
            </div>
           
          </li><br/>
          <li><!--<img id="content" src="http://jb.asm.org/content/196/23/4130/F2.medium.gif" width="60" style="border:1px solid grey;vertical-align:text-bottom">&nbsp;&nbsp;-->D. D. Nayak, C.J. Marx, <a href="http://jb.asm.org/content/196/23/4130.full">Methylamine utilization via the <i>N</i>-methylglutamate pathway in <i>Methylobacterium
            extorquens</i> PA1 involves a novel flow of carbon through C<sub>1</sub> assimilation and dissimilation pathways</a>, <b>J.
Bacteriology</b>, Volume 196, Issue 23, pp 4130–4139, 2014. 
<button type="button" class="btn-xs" data-toggle="collapse" data-target="#demo5"">More Info</button>
            <div id="demo5" class="collapse">
                                            <ul>
                                            <li> Although poorly characterized, the <i>N</i>-methylglutamate (NMG) pathway is the prevalent module for methylamine oxidation in methylotrophs </li>
                                            <li>	Our genetic analyses of the NMG pathway in <i>Methylobacterium extorquens</i> PA1 supports a linear topology for this pathway </li>
<li>	We show that the end-product of the NMG pathway gets dissimilated via the tetrahydromethanopterin (H<sub>4</sub>MPT) linked pathway and directly assimilated via the serine cycle, representing a novel route for C<sub>1</sub> utilization in methylotrophs </li>
<li>	We also demonstrate that formaldehyde activating enzyme (FAE) and its homologs (FAE2, FAE3) have distinct roles during methylamine growth in <i>M. extorquens</i> PA1 </li>
                                            </ul>
            </div>
          </li><br/>
          <li><!--<img id="content" src="http://journals.plos.org/plosone/article/figure/image?size=large&id=10.1371/journal.pone.0107887.g001" width="60" style="border:1px solid grey;vertical-align:text-bottom">&nbsp;&nbsp;--> D. D. Nayak, C.J. Marx, <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0107887">Genotypic and phenotypic comparison of methylotrophy between <i>Methylobacterium</i>
strains AM1 and PA1</a>, <b>PloS One</b>, 9(9): e107887, 2014. <button type="button" class="btn-xs" data-toggle="collapse" data-target="#demo6"">More Info</button>
            <div id="demo6" class="collapse">
              <ul>
              <li>	We report the development of a new model organism for laboratory analyses of the methylotrophy, the ability to grow on reduced single-carbon compounds: <i>Methylobacterium extorquens</i> PA1 </li>
                <li>	We perform genomic analyses to show <i>M. extorquens</i> PA1 has the same repertoire of core methylotrophy genes as <i>M. extorquens</i> AM1 </li>, a  closely-related model strain used for genetic and biochemical analyses of methylotrophy
<li>	We perform genetic analyses to show that core methylotrophy genes in PA1 have the same function characterized in  AM1 </li>
<li>	However, we demonstrate that PA1 grows faster under most conditions and also has a streamlined genome compared to AM1 </li>
<li> Overall, our analyses indicate that PA1 is more suited for laboratory studies, especially long-term evolution experiments </li>
              </ul>
            </div>
          </li><br/>
          <li> B. Boonchayaanant, D. D. Nayak, D. Xin, C. Criddle, <a href="https://www.ncbi.nlm.nih.gov/pubmed/19651424">Uranium Reduction and Resistance to Reoxidation under
Iron-reducing and Sulfate-reducing Conditions</a>, <b>Water Research</b>, Volume 43, Issue 18, pp 4652–4664, 2009
          </li>
          </ul>
          
        </div>
      </div>  
      <div class="row">
        <div class="col-md-12">
          <h2> Manuscripts</h2>
          <ul>
          <li>D. D. Nayak, M. Davlieva, T. Ticak, J. V. Bazurto, J. L. Johnson, F. M. Ytreberg, Y. Shamoo, C. J. Marx, EfgA halts
bacterial translation in response to intracellular formaldehyde. In preparation for <b>Cell</b>.
          </li><br/>
          <li>
          D. D. Nayak, W. W. Metcalf. Methylamine-specific methyltransferases in <i>Methanosarcina spp.</i> are functionality
distinct despite frequent gene conversion. In preparation for <b>PloS Biology</b>.
          </li>

           </ul>
          
        </div>
      </div>  

      <a name="honors"></a>
      <div class="row">
        <div class="col-md-12">
          <h2> Honors and Awards </h2>

           <ul>
           <li>Best Poster Award (1 out of ~100 posters) at the <a href="https://www.grc.org/programs.aspx?id=10900">Gordon Research Conference on Applied and Environmental Microbiology</a> (2017)
           </li>
           <li>
            Best Poster Award (3 out of ~100 posters) at the <a href="https://www.grc.org/programs.aspx?id=12764">Gordon Research Conference on the Molecular Basis of Microbial One-Carbon Metabolism</a> (2016)
          </li>
          <li> <a href="https://www.simonsfoundation.org/life-sciences/simons-postdoctoral-fellowships-in-life-sciences/">Life Sciences Research Foundation Postdoctoral Fellowship</a>, with $50,000 salary and
          $10,000 research expenses per year for three years&mdash;awarded nationally to ~30 from over
            1000 candidates in all fields of life sciences (Aug. 2015&mdash;)
            </li>
            <li> <a href="http://www.igb.illinois.edu/involve/become-a-fellow">Carl R. Woese Institute for Genomic Biology Postdoctoral Fellowship</a> at the University
                of Illinois at Urbana-Champaign, with $50,000 salary and $7,500 research expenses per year for
                three years. (Jan. 2015&mdash;Aug. 2015: salary and research support; 
                Aug. 2015&mdash;: research support only)
                </li>
            <li>
              Certificate of <a href="https://bokcenter.harvard.edu/certificates-excellence-and-distinction-teaching-recipients-fall-2013">Distinction in Teaching 
              for 'Microbial Evolution' from Harvard University</a> (2013)
            </li> 
            <li>Young Investigator Oral Abstract Presentation from the American Society of Microbiology, San Francisco, CA (2013)
            </li>
            <li> The Sigma Xi Grant in Aid-of-Research (2012)
            </li> 
            <li>
            Multiple graduate fellowships from Harvard University (2009) and Stanford University (2007)
            </li>
            </ul>
             </div>
          </div>

      <a name="service"></a>
      <div class="row">
        <div class="col-md-12">
          <h2> Service </h2>
          <p>
          </p>
           <h4 class="media-heading"><img src="https://www.grc.org/graphics/logos/grs_50x35.png" alt="GRS" height="30">&nbsp;&nbsp;&nbsp;<b>Chair and Vice-Chair of Gordon Research Seminars</b></h4>
          <p>
          I served as the Vice-Chair (in <a href="https://www.grc.org/programs.aspx?id=1535">2014</a>) and Chair (in <a href="https://www.grc.org/programs.aspx?id=15360">2016</a>) for the Gordon Research Seminar (GRS) on the ‘Molecular Basis of Microbial One-Carbon Metabolism’. This seminar is held for two days prior to the corresponding Gordon Research Conference (GRC) and is organized by and for early-career scientists. As the Chair, my duties involved screening and selecting ~50 participants from all over the globe, inviting prominent scientists as keynote speakers and discussion leaders, and also acquiring external funding for the meeting. In addition, I developed a mentorship component for the 2016 meeting titled ‘Scientific careers beyond the PhD and postdoc’, which included scientists from industry and academia as well as a senior editor for <i>Science</i>. 
         
          </p>
          <h4 class="media-heading"><img src="http://www.msi.harvard.edu/images/msilogo.gif" alt="MSI" height="30">&nbsp;&nbsp;&nbsp;<b>Microbial Science Initiative Journal Club</b></h4>
          <p>
      As a graduate student at Harvard, I helped establish the <a href="http://www.msi.harvard.edu/events/journalclub.html">Microbial Sciences Initiative (MSI) summer journal club</a>, 
          and was in-charge of running this cross-disciplinary event for its first two years. The MSI journal club is now in its sixth year,
          and continues to attract a number of graduate students and postdocs across campus.
          </p>
        </div>
      </div>    

      
      <a name="outreach"></a>
      <div class="row">
        <div class="col-md-12">
          <h2> Outreach </h2>
          <p>
          I have actively sought out a number of outreach activities as a PhD student and PostDoc. 
          </p>
          <h4 class="media-heading"><b>The Microbiology of Food and Drink: Public Talk and Other Activities</b></h4>
          <p>
          I have designed a broadly accessible talk on the microbiology of food and drink, and delivered it
          at a number of venues, including the <a href="https://www.facebook.com/ChambanaScienceCafe">Chambana Science Cafe</a> in Urbana, the <a href="http://beaconhillseminars.org/">Beacon Hill Seminars</a>,
          and the <a href="http://sitn.hms.harvard.edu/seminar-series/">Science in the News Lecture Series</a> at the Harvard Medical School. I have used this talk as
          a vehicle to introduce some of my work on microbial evolution and physiology.
          Please email me if you'd like to use my slides! I have also collaborated 
           with a cheese monger to culture and showcase the
          fascinating microbiology of cheese at the Cambridge Science Festival. 
          I have also presented a talk titled ‘The role of
          hops and the mechanism of bacterial resistance to hops’ at an MSI (Microbial Science Initiative) workshop.
          </p>
          <h4 class="media-heading"><img src="images/genome-day.png" alt="genome day" height="50">&nbsp;&nbsp;&nbsp;<b>Interactive Activity for Kids in Grades K-4</b></h4>
          <p>
          I designed an activity for the <a href="http://www.igb.illinois.edu/igb-calendar/genome- day-2">Carl R. Woese Institute of Genomic Biology Genome Day</a> highlighting
          the extremophilic nature of archaea, targeted at kids in grades K-4. 
          The activity showcased archaea as superheroes with extremophilic superpowers and the goal was to
          match superheros with an appropriate superpower that will allow them to inhabit various extreme environments on our Planet. 
          If you're a science teacher and would like to use this activity in your class, I would love to speak to you &mdash; please reach out!
          </p>
          
        </div>
      </div>    



      

      
      
        


      
    </div><!--/.container-->



    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
    <!--<script src="../../dist/js/bootstrap.min.js"></script>-->
    <!--<script src="offcanvas.js"></script>-->
  </body>
  </html>


